docs/samples/client/python/oauth/unbranded_client_credential.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any, Optional
+from corehttp.credentials import AccessTokenInfo, TokenRequestOptions
+import time
+import httpx
+
+
+class UnbrandedClientCredential:
```

Comment:
```suggestion
class OAuth2ClientCredential:
```

Don't use the word "unbranded", it's an internal term